browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -1286,4 +1291,78 @@ TEST_F(BraveWalletServiceUnitTest, SignMessage) {
   ASSERT_TRUE(GetPendingSignMessageRequests().empty());
 }
 
+TEST_F(BraveWalletServiceUnitTest, ImportFromMetaMask) {
+  keyring_controller_->Reset();
+
+  BraveWalletServiceDelegate::ImportInfo info;
+  info.mnemonic = kMnemonic1;
+  info.is_legacy_crypto_wallets = false;
+  info.number_of_accounts = 2;
+
+  BraveWalletServiceDelegate::ImportError error = ImportError::kNone;
+  base::RunLoop run_loop;
+  service_->OnGetImportInfo(
```

Comment:
Please use `SimulateOnGetImportInfo` and most of these cases are included in `BraveWalletServiceUnitTest.OnGetImportInfo` so I think you can just update the invalid mnemonic case

Reply:
Sorry rebased and fixed, didn't notice some similar tests already existed.